### ISSUE TITLE
Unhandled error during execution of scheduler flush. - when using TextField's @textChange with style?

### ISSUE THREAD

**Issue by vpiskunov** [nativescript-vue/nativescript-vue#1046]
V3 - After hours of debugging and deducing which exact combination of factors causes the issue, it seems (however weird) that, `style="color: #000000"` in particular, together with `@textChange="onInput"` breaks the app - first results in 
```
[Vue warn]: Unhandled error during execution of scheduler flush. This is likely a Vue internals bug. Please open an issue at https://new-issue.vuejs.org/?repo=vuejs/core
```
on every keystroke inside the TextField, and then a full app crash if trying to navigate away (swipe back) from the page after having edited the field's value:
```
[Vue warn]: Unhandled error during execution of scheduler flush. This is likely a Vue internals bug. Please open an issue at https://new-issue.vuejs.org/?repo=vuejs/core
  at <AuthLayout title="Welcome to XYZ" text="To continue please enter your credentials to logon to the service" backgroundImageSrc="~/assets/hands.png" >
  at <Login>
  ***** Fatal JavaScript exception - application has been terminated. *****
  NativeScript encountered a fatal error: Uncaught TypeError: Cannot read property 'type' of null
  at
  unmountComponent(file: src/webpack:/ma-app/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:6215:50)
  at unmount(file: src/webpack:/ma-app/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:6115:0)
  at unmountChildren(file: src/webpack:/ma-app/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:6259:0)
  at unmount(file: src/webpack:/ma-app/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:6133:0)
  at unmountChildren(file: src/webpack:/ma-app/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:6259:0)
  at unmount(file: src/webpack:/ma-app/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:6133:0)
  at unmountComponent(file: src/webpack:/ma-app/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:6230:0)
  at unmount(file: src/webpack:/ma-app/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:6115:0)
  at unmountComponent(file: src/webpack:/ma-app/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:6230:0)
  at unmount(file: src/webpack:/ma-app/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:6115:0)
  at render(file: src/webpack:/ma-app/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:6274:0)
  at unmount(file: src/webpack:/ma-app/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:4494:0)
  at cleanup(file: src/webpack:/ma-app/node_modules/nativescript-vue/dist/plugins/navigation.js:79:0)
  at targetPage.disposeNativeView(file: src/webpack:/ma-app/node_modules/nativescript-vue/dist/plugins/navigation.js:64:0)
  at _tearDownUI(file: src/webpack:/ma-app/node_modules/@nativescript/core/ui/core/view-base/index.js:661:0)
  at _removeViewCore(file: src/webpack:/ma-app/node_modules/@nativescript/core/ui/core/view-base/index.js:492:0)
  at _removeView(file: src/webpack:/ma-app/node_modules/@nativescript/core/ui/core/view-base/index.js:482:0)
  at _removeEntry(file: src/webpack:/ma-app/node_modules/@nativescript/core/ui/frame/frame-common.js:167:0)
  at _updateBackstack(file:///app/vendor.js<…>
```

<img width="1624" alt="image" src="https://github.com/nativescript-vue/nativescript-vue/assets/6653762/d8ab4d5e-3cd4-44a0-aeaf-73d09d02190f">

**Comment by vpiskunov** [comment]
Additional info: this happens only when `v-bind="{someAttr: 123}"` is on the TextField.

For avoidance of doubt, even with `v-bind="{}"` the issue persists, but this should help someone narrow it down?

```
<TextField
        v-bind="{}"
        :text="credentials.email"
        style="color: #000000"
        @textChange="credentials.email = $event.object.text"
      ></TextField>
```

And credentials is just a ref or reactive of { email: '' } (both reactive and ref have same issue.

Thanks for looking into this!

**Comment by vallemar** [comment]
@vpiskunov I've been following the updates but I don't know what could have fixed it.  If it works with the latest version great!

**Comment by rigor789** [comment]
I believe the fix in #1044 fixed this as well. I saw this happen, and I believe it had to do with a weird edge case around wrapping the native views in a `Proxy` (that vue does internally).
